pom.xml

```diff
@@ -78,6 +78,12 @@
       </dependency>
 
       <dependency>
+        <groupId>commons-io</groupId>
+        <artifactId>commons-io</artifactId>
+        <version>2.4</version>
+      </dependency>
```

Comment:
don't forget that any new dependencies have to be added to modules.xml in both the agent assembly and in the feature pack module.xml